static/app/bootstrap/bootstrapRequests.tsx

```diff
@@ -52,9 +46,23 @@ export function useBootstrapOrganizationQuery(orgSlug: string | null) {
       });
     }
     if (organizationQuery.error) {
-      OrganizationStore.onFetchOrgError(organizationQuery.error as any);
+      if (
+        organizationQuery.error instanceof RequestError &&
+        // By default, `react-query` will refetch stale queries on window focus.
```

Comment:
alternatively or additionally, we could turn off refetch on focus for org details. It's nice to have org context up-to-date, but it likely doesn't update that often?